### DISCUSSION TITLE
Configuring Regions for Vercel Functions

### DISCUSSION THREAD

**Discussion by ohyoxo** [vercel/community#5937] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/126952400/d8619c99-98c7-44a9-87ae-b7da6d508fbb)

I deployed a simple JS test project on Vercel using GitHub Actions, but I have been unable to change the default region despite following the methods provided in the official documentation. Is there any other setting required for this?

### Steps to Reproduce

vercel.json

{
  "regions": ["lhr1"]
}

**Comment by amyegan** [comment, marked as answer]
Hey @ohyoxo! I was able to change the serverless function region for my test app in the [Dashboard](https://vercel.com/docs/functions/configuring-functions/region#dashboard). I also verified that I could override that setting using [`regions`](https://vercel.com/docs/projects/project-configuration#regions) in a vercel.json file.

After selecting a function region, did you click "Save" and create a new deployment? You should see the new region in Functions list of the Build Summary after a new successful deployment.

If you're still having trouble, please share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example). That way someone here may be able to recognize what's going wrong and offer a solution 🙏 

https://vercel.com/docs/functions/configuring-functions/region
